tgui/packages/tgui/interfaces/AntagInfoMalf.tsx

```diff
@@ -101,28 +101,28 @@ function FlavorSection(props) {
           <Stack fill vertical>
             <Stack.Item style={{ backgroundColor: 'black' }}>
               <span style={goalstyle}>
-                System Integrity Report:
+                Отчет о целостности системы:
                 <br />
               </span>
               &gt;{goal}
             </Stack.Item>
             <Stack.Divider />
             <Stack.Item grow style={{ backgroundColor: 'black' }}>
               <span style={allystyle}>
-                Morality Core Report:
+                Доклад ядра морали:
                 <br />
               </span>
               &gt;{allies}
             </Stack.Item>
             <Stack.Divider />
             <Stack.Item style={{ backgroundColor: 'black' }}>
               <span style={badstyle}>
-                Overall Sentience Coherence Grade: FAILING.
+                Общая оценка согласованности чувств: НЕУДАЧА.
```

Comment:
```suggestion
                Общая оценка когерентности осознания: НЕУДАЧА.
```
ИИ получил сознание/возможность решать за себя - это очень плохо по меркам НТ. Можно как-то перефразировать, конечно.